global.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Who updates this file?

Reply:
You can add an arcade subscription and then it would be auto update.